x-pack/plugins/observability/public/pages/rules/global_logs_tab.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { RULE_DETAIL_PATH } from '../../../common/locators/paths';
+import { useGetFilteredRuleTypes } from '../../hooks/use_get_filtered_rule_types';
+import { useKibana } from '../../utils/kibana_react';
+
+export function GlobalLogsTab() {
+  const {
+    triggersActionsUi: { getGlobalRuleEventLogList: GlobalRuleEventLogList },
+  } = useKibana().services;
+  const filteredRuleTypes = useGetFilteredRuleTypes();
+
+  const getRuleDetailsRoute = (ruleId: string) => RULE_DETAIL_PATH.replace(':ruleId', ruleId);
```

Comment:
You can use paths for this purpose: ([example](https://github.com/elastic/kibana/pull/163091/files#diff-152157b936cad35c1aef23f8536b3c71ffb2be7b6f67cd955b51ad7df15e117fR185))

```
import { paths } from '../../../../common/locators/paths';

...

paths.observability.ruleDetails(rule.id)
```

Reply:
Looks like it's not the same. I need `/alerts/rules/c0b230c0-47ff-11ee-9247-0db8120e57cd` but `/app/observability/alerts/rules/c0b230c0-47ff-11ee-9247-0db8120e57cd` is generated with paths.observability.ruleDetails(rule.id)

Reply:
I see. 

nit: I don't see it as a blocker, but replacing `RULE_DETAIL_PATH` with ruleId seems a bit fragile (if we change `:ruleId` for some reason, we will break this link.)
Maybe we can also export `relativePaths` from `common/locators/paths` to export paths without the app prefix 🤔